### ISSUE TITLE
--inferoptions returns options which are deprecated

### ISSUE THREAD

**Issue by mrackwitz** [nicklockwood/SwiftFormat#227]
This causes the following warning when using the options to format the code:
```
`--insertlines` option is deprecated. Use `--disable blankLinesBetweenScopes` or `--disable blankLinesAroundMark` instead
`--removelines` option is deprecated. Use `--disable blankLinesAtStartOfScope` or `--disable blankLinesAtEndOfScope` instead
```

**Comment by nicklockwood** [comment]
@mrackwitz thanks, I'll fix it.

**Comment by nicklockwood** [comment]
Fixed in 0.33.3
